client/state/sites/plans/selectors.js

```diff
@@ -201,11 +200,6 @@ export function getSitePlanSlug( state, siteId ) {
 	return get( getCurrentPlan( state, siteId ), 'productSlug', null );
 }
 
-// Duplicated from lib/plans. Proper solution in https://github.com/Automattic/wp-calypso/pull/9635
-function planHasFeature( plan, feature ) {
-	return includes( get( PLANS_LIST[ plan ], 'getFeatures', () => [] )(), feature );
-}
-
```

Comment:
Replaced with new implementation of `planHasFeature` from `client/lib/plans/index.js`, as was done in https://github.com/Automattic/wp-calypso/pull/9635.